### ISSUE TITLE
[Suggestion] Community Topic Tribunal System

### ISSUE THREAD

**Issue by TacticalTriangle** [Backdash/MonikaModDev#616]
So i was thinking about the daily Monika's Writing Tip of the Day #324 and how that would actually be implemented and an idea came to my head. What if there was a type of community based topic submission system? Unlike the system that is being put into place, This one would require the person submitting it to write it out like we used to in the PR system but instead it would go through the community to decide whether or not it is a good topic. Then go through some other moderator(not dev team) then finally it gets brought to the dev team. This type of system would A) Make things like daily Writing Tips more plausible with the new system being put in place and B) Give the community more opportunities to contribute to the mod in a way similar to what they were used to. It can also(if needed) show writers who have frequently gotten their topics through so that if it ever is needed to add more writers to the core team or the moderators in charge of the tribunal system.

TLDR;
Community submits written topics to Community>It then gets to go through some moderators(Not Dev Team). Then Dev Team. To make more topics to make things like #324 possible while A) not drowning the dev team in things to look over and B) giving the community more room to contribute other then the topic submissions system that is taking over.

**Comment by jukebox1412** [comment]
I believe this has already been suggested by #188.
